contracts/fast/FastFrontendFacet.sol

```diff
@@ -8,10 +8,6 @@ import './lib/LibFastAccess.sol';
 import './lib/LibFastToken.sol';
 
 
-/** @title The SPC Smart Contract.
```

Comment:
I noticed that lots of these were copy-pasted. I'll do a pass on documentation later.